app/views/chapters/show.html.erb

```diff
@@ -1,5 +1,9 @@
+<%= content_for :breadcrumbs do %>
+  <%= path_breadcrumb @chapter %>
+<% end %>
+
 <div>
-  <h1><%= @chapter.contextualized_name %></h1>
+  <h1><%= t(:chapter_number, number: @chapter.number) %> <%= @chapter.name %></h1>
```

Comment:
Perhaps we could extract this logic to a helper

`t(:#{object.model_name}_number, number: #{object}.number) %> <%= #{object}.name %>`